src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_word_2012_wordml.g.cs

```diff
@@ -1082,7 +1082,6 @@ internal override void ConfigureMetadata(ElementMetadata.Builder builder)
 })
 .AddAttribute(69, "contact", a => a.Contact, aBuilder =>
 {
-aBuilder.AddValidator(RequiredValidator.Instance);
```

Comment:
This is in a generated filed. The generator needs to be updated.

Reply:
I haven't pushed it yet.

Reply:
Now it's there.

Reply:
I added the obsolete contact property and removed (genned out) it in the generated code.